src/status_im/wallet_connect_legacy/core.cljs

```diff
@@ -51,7 +51,7 @@
   (let [proposal (js->clj request-event :keywordize-keys true)
         params (first (:params proposal))
         metadata (merge (:peerMeta params) {:wc-version 1})
-        chain-id (:chainId params)]
+        chain-id @(re-frame/subscribe [:chain-id])]
```

Comment:
subscription can't be used in event you have to get data from db

Reply:
Updated, thanks!